fragment_internal_test.go

```diff
@@ -213,39 +213,6 @@ func TestFragment_SetValue(t *testing.T) {
 			t.Fatal(err)
 		}
 	})
-	t.Run("Crash", func(t *testing.T) {
```

Comment:
todd added this test when trying to fix that crash bug - it didn't turn out to be useful, so I've removed it here.